test/integration/Psr/SimpleCache/MemcacheIntegrationTest.php

```diff
@@ -67,6 +59,8 @@ public function createSimpleCache(): SimpleCacheDecorator
             $options['servers'] = [[$host]];
         }
 
+        $this->skippedTests['testBasicUsageWithLongKey'] = 'SimpleCacheDecorator requires keys to be <= 64 chars';
```

Comment:
Why is there an integration test for long keys if they're not supported by the PSR standard?
Or does memcache limit the cache key to `<= 64 chars`?
If not, we should not skip the test but fix the problem which is causing the issue?

Reply:
Nothing to do with `memcache`, here the sources:

1. https://github.com/php-cache/integration-tests/pull/86/files#diff-202e22f7f977d4ea227f5b444dbdd19485fb8093696b949dd4b918930c7a6e46R116
2. https://github.com/laminas/laminas-cache/commit/5e10ef6222ff342e651387630ea1637aa7c68e29

Reply:
Hmm, I am torn.
The PSR states that every PSR-16 implementing cache **must** support 64 chars. It **can** support more than that.
So imho, https://github.com/laminas/laminas-cache/commit/5e10ef6222ff342e651387630ea1637aa7c68e29 was based on a wrong assumption on how PSR-16 works.

I am fine with this for the 1.x versions.
Imho, having a new required option for the `AdapterOption` for v3 would help us optimize the decorators.

So I'll create a new Issue for that.

Reply:
https://github.com/laminas/laminas-cache/issues/91